CHANGELOG.md

```diff
@@ -0,0 +1,21 @@
+# Changelog
```

Comment:
Should this be updated with details about the initial release?

Reply:
Yeah, it will be updated when I know when we're releasing this. Right before merging this to master.